components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/AccessTokenIssuer.java

```diff
@@ -400,6 +473,13 @@ public OAuth2AccessTokenRespDTO issue(OAuth2AccessTokenReqDTO tokenReqDTO)
             log.debug("Access token issued to client Id: " + tokenReqDTO.getClientId() + " username: " +
                     tokReqMsgCtx.getAuthorizedUser() + " and scopes: " + tokenRespDTO.getAuthorizedScopes());
         }
+        Map<String, Object> params = new HashMap<>();
```

Comment:
move the map creation to the if block, do the same at other places as well

Reply:
+1. addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1737/commits/b11a186a08caa35286029e46bba6f115aa4ce711